views/files/files-grid.hbs

```diff
@@ -26,43 +28,53 @@
 								{{#ifeq this.securityCheck.status "blocked"}}
 									<a href="#"
 									class="btn-file-danger"
-									title="{{$t "files.label.dangerousCode"}}">
+									title="{{$t "files.label.dangerousCode"}}"
+									aria-label="{{$t "files.label.dangerousCode"}}">
 										<i class="fa fa-exclamation-triangle"></i>
 									</a>
 								{{else}}
 									<a href="/files/file?download=true&file={{../this._id}}&name={{../this.storageFileName}}"
 										target="_blank"
 										data-method="download"
-										title="{{$t "files.label.downloadFile"}}">
+										title="{{$t "files.label.downloadFile"}}"
```

Comment:
same here?  :)

Reply:
It is always good to add a warning for screen reader users when they are redirected to a new page/tab, otherwise they might expect to be still on the same page. So here I would add the warning to the aria-label (the title doesn't need that warning obviously).